pb-commit-message-lints/src/lints/lib/commit_message.rs

```diff
@@ -242,18 +295,175 @@ mod test_commit_message {
             ),
             CommitMessage::new(
                 indoc!(
```

Comment:
Do you think it's worth it if I (in another commit) switch to using indoc? I wasn't sure if it was idiomatic

Reply:
I'm not sure if indoc is idiomatic but I prefer it to breaking the indentation in these sort of tests.

Reply:
I'll have a play with it for some of the other test